spec/controllers/users/totp_setup_controller_spec.rb

```diff
@@ -32,10 +30,6 @@
           subject.current_user.decorate.qrcode(subject.user_session[:new_totp_secret])
         ).not_to be_nil
       end
-
-      it 'presents a QR code to the user' do
```

Comment:
Agree we should have stuff covered by feature specs -- what specifically about the TOTP setup should we have covered by feature specs? I could create one now or via a separate GH issue. 

Right now, as far as I can tell, the only place we visit `'/authenticator_setup'` in feature specs is in the accessiblity specs.